src/components/App/container.js

```diff
@@ -0,0 +1,12 @@
+import {connect} from 'react-redux';
+import App from './App';
+import {setSendWithPhoto} from './actions';
+
+const mapDispatchToProps = dispatch => ({
+	setSendWithPhoto: sendWithPhoto => dispatch(setSendWithPhoto(sendWithPhoto))
```

Comment:
Could use shorthand notation, we'll change this in a different piece of work for now though

Reply:
fixed this in a later commit